### PR TITLE
Update README.md - add description

### PR DESCRIPTION
**Pull request by 3esmit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Status.im Sticker Market Dapp
 
+Sticker Market is composed of 3 contracts: 
+- An authorship NFT (StickerType) which is granted to every registered StickerPack in StickerMarket, intended use is to change the deposit address for all StickerPacks sold in StickerMarket.
```
Comment (0x-r4bbit) [inline README.md]:
```suggestion
- An authorship NFT (`StickerType`) which is granted to every registered `StickerPack` in `StickerMarket`, intended use is to change the deposit address for all `StickerPack`s sold in `StickerMarket`.
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+- StickerMarket main contract, which is the minter of two other NFT smart contract addresses, StickerType and StickerPack. It allows users to mint StickerType, which contains a content-hash link, and other users to mint StickerPack referenced to certain StickerTypes.
```
Comment (0x-r4bbit) [inline README.md]:
```suggestion
- `StickerMarket` main contract, which is the minter of two other NFT smart contract addresses, `StickerType` and `StickerPack`. It allows users to mint `StickerType`, which contains a content-hash link, and other users to mint `StickerPack` referenced to certain `StickerType`s. 
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Status.im Sticker Market Dapp
 
+Sticker Market is composed of 3 contracts: 
+- An authorship NFT (StickerType) which is granted to every registered StickerPack in StickerMarket, intended use is to change the deposit address for all StickerPacks sold in StickerMarket.
+- StickerMarket main contract, which is the minter of two other NFT smart contract addresses, StickerType and StickerPack. It allows users to mint StickerType, which contains a content-hash link, and other users to mint StickerPack referenced to certain StickerTypes. 
+- A purchase NFT (StickerPack) referencing a StickerType, which is granted to any account who paid the value defined in the StickerType while it sale was available.
```
Comment (0x-r4bbit) [inline README.md]:
```suggestion
- A purchase NFT (`StickerPack`) referencing a `StickerType`, which is granted to any account who paid the value defined in the `StickerType` while it sale was available. 
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 # Status.im Sticker Market Dapp
 
+Sticker Market is composed of 3 contracts: 
+- An authorship NFT (StickerType) which is granted to every registered StickerPack in StickerMarket, intended use is to change the deposit address for all StickerPacks sold in StickerMarket.
+- StickerMarket main contract, which is the minter of two other NFT smart contract addresses, StickerType and StickerPack. It allows users to mint StickerType, which contains a content-hash link, and other users to mint StickerPack referenced to certain StickerTypes. 
+- A purchase NFT (StickerPack) referencing a StickerType, which is granted to any account who paid the value defined in the StickerType while it sale was available. 
+
+Therefore, it models an envoirment where artists can publish their packs for sale, that can be used in official Status app, by holders of StickerPack.  Artists can also sell their token, which new owner could set a new payment address.
```
Comment (0x-r4bbit) [inline README.md]:
```suggestion
Therefore, it models an environment where artists can publish their packs for sale, that can be used in the official Status app, by holders of `StickerPack`.  Artists can also sell their tokens, which new owners could set a new payment address.
```

**Review by 0x-r4bbit — CHANGES_REQUESTED** [review]
Nice, thank you @3esmit !
I've suggested some minor fixes and formatting.

Maybe you can update the PR accordingly.

**Comment by 3esmit** [comment]
Good improvements. Commited them.

**Review by 0x-r4bbit — APPROVED** [review]

**Comment by 0x-r4bbit** [comment]
Good job!

**Pull request merged**